Disable the updater and related stuff.  Does nothing under OpenBSD

Encrypted Media Extensions is not supported on OpenBSD. Apparently this also
means we can not pass --disable-eme to configure so comment that out.

Don't use --enable-rust-simd, fails to build.

Index: .mozconfig
--- .mozconfig.orig
+++ .mozconfig
@@ -14,7 +14,6 @@ mk_add_options MOZ_APP_DISPLAYNAME="Tor Browser"
 export MOZILLA_OFFICIAL=1
 
 ac_add_options --enable-optimize
-ac_add_options --enable-rust-simd
 ac_add_options --enable-official-branding
 
 # Let's support GTK3 for ESR60
@@ -28,13 +27,15 @@ ac_add_options --disable-crashreporter
 ac_add_options --disable-webrtc
 ac_add_options --disable-parental-controls
 # Let's make sure no preference is enabling either Adobe's or Google's CDM.
-ac_add_options --disable-eme
+#ac_add_options --disable-eme
 ac_add_options --enable-proxy-bypass-protection
 
 # Disable telemetry
 ac_add_options MOZ_TELEMETRY_REPORTING=
 
-ac_add_options --disable-tor-launcher
-ac_add_options --with-tor-browser-version=dev-build
+# avoid 1.1GB libxul
+ac_add_options --disable-debug-symbols
+
+#ac_add_options --with-tor-browser-version=dev-build
 ac_add_options --disable-tor-browser-update
-ac_add_options --enable-verify-mar
+#ac_add_options --enable-verify-mar
